frontend/src/breeding_form.tsx

```diff
@@ -68,10 +69,49 @@ export function BreedingForm({
   const { userMessage } = useMessageContext();
   const [formState, setFormState] = React.useState(emptyBreeding as Breeding);
   const [showBirthForm, setShowBirthForm] = React.useState(false);
+  let defaultDate = new Date();
+  defaultDate.setFullYear(defaultDate.getFullYear() - 10);
+  const [fromDate, setFromDate] = React.useState(defaultDate as Date);
+  const [activeMalesLimited, setActiveMalesLimited] = React.useState([]);
+  const [activeFemalesLimited, setActiveFemalesLimited] = React.useState([]);
+  const { user } = useUserContext();
+  const is_admin = !!(user?.is_manager || user?.is_admin);
+  const is_owner = !!(user?.is_owner && user.is_owner.length > 0);
+
+  const genebank: Genebank | undefined = React.useMemo(() => {
+    return genebanks.find((g) => g.herds.find((h) => h.herd == herdId));
+  }, [genebanks]);
+
   React.useEffect(
     () => setFormState(!data || data == "new" ? emptyBreeding : data),
     [data]
   );
+
+  React.useEffect(() => {
+    setActiveFemalesLimited(
+      toLimitedIndividuals(
+        getIndividuals(
+          "female",
+          is_admin || is_owner,
```

Comment:
So with the definition of `is_admin` and `is_owner` above, this line only returns `false` if a user is not admin, not genebank manager and does not own a herd. That seems odd to me. Do we even have these users? And why would we have a limitation for them here?